templateutils/example_test.go

```diff
@@ -50,7 +50,6 @@ func ExampleGenerateUsageUndecorated() {
 	i := struct {
 		X       int
 		Y       string
-		hidden  float64
```

Comment:
We're going to need to skip this file.  This field is hidden for a reason.  To demonstrate that it is excluded from the auto generated documentation.